src/main/java/com/github/manolo8/darkbot/core/manager/EffectManager.java

```diff
@@ -60,15 +60,132 @@ public enum Effect {
         UNDEFINED(-1),
         LOCATOR(1),
         PET_SPAWN(2),
+        SOLACE_NANO_CLUSTER(3),
+        DIMINISHER_WEAKEN_SHIELDS(4),
+        SPECTRUM_PRISMATIC_SHIELD(5),
+        SENTINEL_FORTRESS(6),
+        /** Cyborg uses the same effect as venom for singularity */
+        VENOM_SINGULARITY(7),
+        SABOTEUR(9),
+        /* SHIELD_1(10), */
         ENERGY_LEECH(11),
+        REPAIR_BATTLE_BOT(12),
+        /* LEVEL_UP(13), */
+        LIGHTNING_ABILITY(14),
+        HEAL_RAY(15), // AEGIS_HP_REPAIR RAY
         NPC_ISH(16),
-        DRAW_FIRE(36),
+        /* RAGE(17),
+        SKULL(18),
+        SPAWN(19), */
+        BOX_COLLECTING(20),
+        BOOTY_COLLECTING(21),
+        /* STATIC_MAP(22),
+        ACHIEVEMENT(23),
+        SPAWN_CHILDREN(24),
+        IS_SPAWNED(25),
+        UBER(26),
+        ABSORBATION(27),
+        ABSORBATION_ELITE(28), */
+        LEONOV_HOME_BONUS(29),
+        SHIELD_ENGINEERING_SKILL(30),
+        CHARGE_SHIELD_RAY(31),
+        /* MARKED_BY_BEACON(32),
+        STANDALONE_RING_POD(33),
+        DAMAGE_TRANSFER_RAY(34), */
+        CITADEL_FORTIFY(35),
+        CITADEL_DRAW_FIRE(36),
+        CITADEL_PROTECTION(37),
+        PROTECTED(38),
```

Comment:
```suggestion
        /** A nearby citadel is casting protection on this ship */
        CITADEL_PROTECTED(38),
```
I'd still leave the citadel prefix as that's what triggers it, otherwise "protected" on its own has little meaning